app/domain/authentication/authenticate.rb

```diff
@@ -4,7 +4,6 @@
 
 module Authentication
 
-  Err ||= Errors::Authentication
```

Comment:
Found the following as part of the design for authn-azure and azuthn-status

```
    Err = Errors::Authentication::AuthnAzure
```

```
    Err = Errors::Authentication::Authn<Type>
```

Should we update those too?

Reply:
along with

```
 # Possible Errors Raised:
```

Reply:
Nah, i don't think we need to update old designs when we perform refactors